src/components/time-input/TimeInput.tsx

```diff
@@ -27,33 +34,40 @@ export interface TimeInputProps extends WrappedComponentProps {
     label?: React.ReactNode;
     /** initialDate - Date object for initializing the time input */
     initialDate?: Date;
+    /** innerRef - Ref for the time input */
+    innerRef?: React.Ref<HTMLInputElement>;
     /** isRequired - Whether the time input is required */
     isRequired?: boolean;
     /**
      * onBlur - Function to call when the user blurs out of the time input
      * The parsed display time, along with the hours and minutes in 24-hour format, will be passed to the handler.
      */
     onBlur?: ({ displayTime, hours, minutes }: { displayTime: string; hours: number; minutes: number }) => void;
-    /** onChange - Function to call when the user edits the time input */
-    onChange?: (value: string) => void;
+    /** shouldAutoBlur - Whether the field should automatically blur when form submit keys are pressed */
```

Comment:
Since it's not changing the focus target, I think this would be like "format or validate on <something>" rather than blur?

Reply:
Bringing back `onChange` but debounced + with this type of auto validation could also make sense

Reply:
Thanks for the suggestions, @alexkrolick. I updated the PR to include the debounced formatting function -- could you review again?